src/test/java/edu/harvard/iq/dataverse/license/dto/LicenseMapperTest.java

```diff
@@ -0,0 +1,72 @@
+package edu.harvard.iq.dataverse.license.dto;
+
+import com.google.common.collect.Lists;
+import edu.harvard.iq.dataverse.license.License;
+import edu.harvard.iq.dataverse.license.LicenseIcon;
+import edu.harvard.iq.dataverse.license.LocaleText;
+import io.vavr.control.Try;
+import org.apache.commons.io.IOUtils;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Locale;
+
+class LicenseMapperTest {
```

Comment:
In other projects we decided for public test classes and public test methods. We can talk further about this face to face.